digest/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "digest"
```

Comment:
I think the standard approach for proc macros with additional dependencies is to re-export the proc-macro from the other crate so people only have to depend on one crate.